src/gurt/telemetry.c

```diff
@@ -1221,12 +1192,16 @@ d_tm_mark_duration_start(struct d_tm_node_t **metric, int clk_id,
  * does not already exist.
  *
  * \param[in]		metric	Pointer to the metric
- * \param[in]		item	Full path name to this \a item
- *				If supplied, the path name is specified by the
- *				\a item and any strings following it.  The last
- *				item must be NULL.
- *
- * \return			D_TM_SUCCESS		Success
+ * \param[in]		err	If non-zero, aborts the interval calculation
+ *				so that this interval is not added to the stats.
```

Comment:
This param seems a little odd to me. Is it really needed? It seems like the caller could check this value before calling this method

Reply:
It was a special ask by @johannlombardi to add that.

Reply:
Ah, okay. I'm curious about his motivation, I guess probably to limit the call to one line?